app.go

```diff
@@ -60,7 +61,8 @@ func Loop(args []string, onReady func(*App)) error {
 	defer cerr.free()
 
 	app := App{
-		ready: make(chan struct{}),
+		ready:  make(chan struct{}),
+		server: newServer(http.DefaultServeMux),
```

Comment:
I see where you're coming from, but I still think that the http server should be decoupled from window management. The reason, I guess, is that it doesn't seem like the benefits are large enough to make it worth having them coupled. There definitely are benefits to having them coupled, but I just don't think those benefits are large enough. It should be fairly painless for the user to construct full URLs, and creating your own serve mux is a fairly standard thing that most go developers who have build web apps do often.